vita3k/modules/SceDriverUser/SceDisplayUser.cpp

```diff
@@ -23,6 +23,8 @@ TRACY_MODULE_NAME(SceDisplayUser);
 
 EXPORT(SceInt32, sceDisplayGetFrameBuf, SceDisplayFrameBuf *pFrameBuf, SceDisplaySetBufSync sync) {
     TRACY_FUNC(sceDisplayGetFrameBuf, pFrameBuf, sync);
+    if (!pFrameBuf)
+        return RET_ERROR(SCE_DISPLAY_ERROR_INVALID_VALUE); // is this even correct??, its better than crashing ig
```

Comment:
from disasm ksceDisplayGetFrameBuf (before there is a lot of bypassing values as is)
```  if (pFrameBuf== (SceDisplayFrameBuf *)0x0) {
    return SCE_DISPLAY_ERROR_INVALID_ADDR;
  }
  if ((pFrameBuf->size & 0xfffffffb) != 0x18) {
    return SCE_DISPLAY_ERROR_INVALID_VALUE;
  }```

Reply:
should be on par now :D